trollmoves/server.py

```diff
@@ -418,32 +463,29 @@ def process_notify(orig_pathname, publisher, pattern, attrs):
     """Publish what we have."""
     if not fnmatch.fnmatch(orig_pathname, pattern):
         return
-    elif (os.stat(orig_pathname).st_size == 0):
-        # Want to avoid files with size 0.
+    elif os.stat(orig_pathname).st_size == 0:
         LOGGER.debug("Ignoring empty file: %s", orig_pathname)
         return
     else:
         LOGGER.debug('We have a match: %s', orig_pathname)
 
     pathname = unpack(orig_pathname, **attrs)
+    info = _get_inotify_message_info(attrs, orig_pathname, pathname)
```

Comment:
I'm a bit confused by the function name here, but maybe I'm not really following: what does inotify have to do in the posttroll notifier function?

Reply:
Umm, the `_get_inotify_message_info()` is called in `process_notify()` function which is called only in the nested function `process_notify_publish()` of `create_inotify_notifier()`. So it is part of the inotify chain :-) I'll adjust the ordering of the functions to make this more clear.

Reply:
Oh, you were right. It is used as a callback (sigh) in all the `create_*_notifier()` functions.